src/components/SkipLink.tsx

```diff
@@ -9,9 +9,11 @@ const StyledSkipLink = styled(Link)`
   align-items: center;
   height: 3rem;
   background: ${({ theme }) => theme.palette.section.black};
+  clip-path: circle(0%);
```

Comment:
what? hva betyr dette? at størrelsen på sirkelen er 0? hvorfor er det nødvendig med denne? 😅

Reply:
Det er for at den skal bli skjult når den ikke har fokus :)
en slags `display: none`, bare at den fortsatt kan få fokus. Her sier vi bare at linken skal "clippes" til en sirkel med størrelse 0% (altså ingenting) i forhold til egentlig størrelse.
Du lurer kanskje på hvorfor det trengs i link til skjema, men ikke link til main? jeg også :)

kan egentlig anbefale å ta ned branchen og leke litt - skjer mye rart når man endrer en liten ting